source/wp-content/themes/wporg-parent-2021/sass/base/_elements.scss

```diff
@@ -28,6 +28,22 @@ mark.has-inline-color {
 	}
 }
 
+// For links that navigate users to different site
+.external-link > a::after, // In case we don't access to <a> tag
+a.external-link::after {
```

Comment:
Instead of a custom class, could you use attribute selection to check if the `href` does not contain `wordpress.org`? I think this would work `a:not([href*="wordpress.org"])` ?

Reply:
Yep, this appears to work on showcase (and a few other tests).

Reply:
It's important to note that this is aggressive :)

**Footer**
<img width="381" alt="Screen Shot 2022-11-17 at 12 35 33 PM" src="https://user-images.githubusercontent.com/1657336/202348792-b544e395-8695-4532-bfa2-c0e3863c9ebb.png">

I'll need to run a more in-depth test before merging. Switching this back to a draft until I can.

Reply:
After testing this patch using `wporg-main`, I think I'm tempted to move back to having an `external` class. I think we'll end up with a lot of places that have external URLs that we don't want to have the icon. 

<img width="486" alt="Screen Shot 2022-11-22 at 3 59 11 PM" src="https://user-images.githubusercontent.com/1657336/203248353-dc85d5ae-7df2-4fb7-a693-2dda1d55757a.png">
<img width="557" alt="Screen Shot 2022-11-22 at 4 16 38 PM" src="https://user-images.githubusercontent.com/1657336/203249470-e889a119-d519-4f91-bb5f-9a6b8913fe55.png">

Reply:
This is why I added a second `CSS` line to the declaration, `.external-link > a::after,` so it can be added to the parent. Not super excited about that but at least it works?

https://github.com/WordPress/wporg-parent-2021/pull/53/commits/1723230ccce77a0a8cbfdb2c41c4e1fc41c4923e#diff-dc9ae95ab0a8f0d1179f51200575b6479f49df7f5517f1afeef40a86aa475d09R32

Reply:
> I think you'll need some kind of has-external-link class that can be added via Advanced (no need to make a block style), but then also combine it with the :not(…) code to avoid accidentally marking internal links if there are multiple in a paragraph, for example.

Ah originally misread that. Yeah, multiple links is an issue. Thanks for flagging that.